br/pkg/task/restore_test.go

```diff
@@ -44,12 +55,11 @@ func TestconfigureRestoreClient(t *testing.T) {
 		RestoreCommonConfig: restoreComCfg,
 		DdlBatchSize:        128,
 	}
-	client := &restore.Client{}
-
+	client := restore.NewRestoreClient(mockPDClient{}, nil, keepalive.ClientParameters{}, false)
 	ctx := context.Background()
 	err := configureRestoreClient(ctx, client, restoreCfg)
 	require.NoError(t, err)
-	require.Equal(t, client.GetBatchDdlSize(), 128)
+	require.Equal(t, uint(128), client.GetBatchDdlSize())
 	require.True(t, true, client.IsOnline())
```

Comment:
it will be always Passing.

Reply:
after remove all `-//br/pkg/task:task_test` in `Makefile`, it will run this case.